stablehlo/tests/stablehlo_probe_instrumentation.mlir

```diff
@@ -0,0 +1,75 @@
+// RUN: stablehlo-opt --stablehlo-instrument --split-input-file --verify-diagnostics %s | FileCheck %s
+
+// CHECK-LABEL: func @instrument_basic_no_location
+func.func @instrument_basic_no_location(%arg0: tensor<1x2xi32>, %arg1: tensor<1x2xi32>) -> tensor<1x2xi32> {
+  // CHECK: [[RESULT:%.*]] = interpreter.probe %0, probe_id = "1" : tensor<1x2xi32>
```

Comment:
I'm wondering about the usability of this default `probe_id`. Do you think there will be a need to `ctrl+f` default `probe_id`? If so, prefixing it with `id_` like `id_1` may be an option to distinguish them from other digits in the exported file.

Maybe it's not necessary if we can `ctrl+f` the `probe_id` field instead. Thoughts?

Reply:
Hm, I don't have a strong preference either way. But maybe changing it to `probe1` vs `probe` in this case better aligns with the proposed changes in #1840?

Reply:
Sounds good!